.github/workflows/slack-send-message.yml

```diff
@@ -4,14 +4,14 @@ on:
     inputs:
       channel-or-user:
         description: 'Send a message to a channel or a user (prefer usage of id instead of name).'
-        # C042KNPSN7N is a test channel
-        # CSF7TG6N5 is the production channel
-        default: 'C042KNPSN7N'
+        # C02J5M4JMK7 is a test channel
+        # vars.SLACK_CHANNEL_ID_NOTIFICATIONS is the production channel
+        default: 'C02J5M4JMK7'
```

Comment:
As this repository is public, I suggest having no default value, and leave it required, so it is the responsibility of the caller to ensure to point to an existing channel.
Unless the pointed Slack channel is also a public Slack channel

Reply:
The caller is a public repo. So, it's also visible.  
We can put it in a `vars` as it's done for the production channel :smile:

Reply:
@educhastenier thanks for the feedback.

This workflow is only intending to test that we correctly configure the slack action and that the messages are correctly posted.
It is only run on demand by contributors with write permissions on the repository.
We don't want to have to remember the id of the channel, so we are going to continue to have this default value.

Reply:
Fine to me